superx/models.py

```diff
@@ -0,0 +1,43 @@
+"""Data models."""
+from . import db
+
+
+class User(db.Model):
+    """Data model for user accounts."""
+
+    __tablename__ = 'item_info'
```

Comment:
lets plan a design session for the database. This needs some work

Reply:
i understand that you did a session with yoav, i will sit with him later and try to make progress